dot/state/block_notify_test.go

```diff
@@ -33,11 +33,10 @@ var testMessageTimeout = time.Second * 3
 func TestImportChannel(t *testing.T) {
 	bs := newTestBlockState(t, testGenesisHeader)
 
-	ch := make(chan *types.Block, 3)
-	id, err := bs.RegisterImportedChannel(ch)
+	ch, err := bs.GetImportedBlockNotifierChannel()
 	require.NoError(t, err)
 
-	defer bs.UnregisterImportedChannel(id)
+	defer bs.FreeImportedBlockNotifierChannel(ch)
```

Comment:
can you confirm the channel is removed from the map after this?

Reply:
double checking on this

Reply:
Yes, I've added a test to confirm this.